emily/handler/src/database/accessors.rs

```diff
@@ -480,6 +574,60 @@ async fn query_with_partition_key<T: TableIndexTrait>(
     .await
 }
 
+async fn query_all_with_partition_and_sort_key<T: TableIndexTrait>(
+    context: &EmilyContext,
+    parition_key: &<<<T as TableIndexTrait>::Entry as EntryTrait>::Key as KeyTrait>::PartitionKey,
+    sort_key: &<<<T as TableIndexTrait>::Entry as EntryTrait>::Key as KeyTrait>::SortKey,
+    sort_key_operator: &str,
+    maybe_page_size: Option<i32>,
+) -> Result<Vec<<T as TableIndexTrait>::Entry>, Error> {
+    // item aggregator.
+    let mut items: Vec<<T as TableIndexTrait>::Entry> = Vec::new();
+    // Next token.
+    let mut next_token: Option<String> = None;
+    // Loop over all items.
+    loop {
```

Comment:
I'm usually a little wary of loops. This is definitely the right thing to so, but we might want to have some other break condition if this is taking too long.

Actually, maybe the lambda timeout is that break condition so this isn't that bad. Also, we're reaching out to dynamodb so this very likely isn't terrible either. Never mind.